src/main/java/org/folio/circulation/support/http/client/OkapiHttpClient.java

```diff
@@ -56,7 +56,7 @@ public void post(
 
     addStandardHeaders(request);
 
-    request.setTimeout(5000);
+    request.setTimeout(15000);
```

Comment:
Why has this timeout been increased by such a degree?

Reply:
I had to increase them to run/debug locally because my laptop is slow or when I need to pause for debug, and I accidentally checked these high timeouts in. I will revert these timeouts.